### PR TITLE
ENG-2318 - Add create application deep link on guides

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

Adds deep links to Customer Portal Create Application page on guides

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/guides/encrypt_us_bank_accounts/index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ app.listen(port, async () => {
 
 <span class="base-alert warning">
   <span>
-    To start, you'll need a new Server to Server Application with the <code>token:bank:create</code> and <code>token:bank:read:high</code> permission
+    To start, you'll need a new Server to Server Application with the <code>token:bank:create</code> and <code>token:bank:read:high</code> permission. <a href="https://portal.basistheory.com/applications/create?type=server_to_server&permissions=token%3Abank%3Acreate&permissions=token%3Abank%3Aread%3Ahigh&name=PII+Retriever" target="_blank">Click here to create one.</a>
```
Comment (koalaty-code) [inline collections/guides/encrypt_us_bank_accounts/index.md:103]:
```suggestion
    To start, you'll need a new Server to Server Application with the <code>token:bank:create</code> and <code>token:bank:read:high</code> permission. <a href="https://portal.basistheory.com/applications/create?type=server_to_server&permissions=token%3Abank%3Acreate&permissions=token%3Abank%3Aread%3Ahigh&name=Bank+Retriever" target="_blank">Click here to create one.</a>
```

File: collections/guides/collect_pii_react/index.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ If you'd like to follow along with this guide from scratch, we suggest creating
 
 <span class="base-alert warning">
   <span>
-    To start, you'll need a <code>Elements</code> [Application](https://docs.basistheory.com/api-reference/#applications) with the <code>token:pii:create</code> permission.
+    To start, you'll need a <code>Elements</code> [Application](https://docs.basistheory.com/api-reference/#applications) with the <code>token:pii:create</code> permission. <a href="https://portal.basistheory.com/applications/create?permissions=token%3Apii%3Acreate&type=elements&name=PII+Collector" target="_blank">Click here to create one.</a>
```
Comment (koalaty-code) [inline collections/guides/collect_pii_react/index.md:25]: This deep link is also missing the application type. Do we want to include it so we aren't dependent on the default selection?

File: collections/guides/collect_charge_cards_js/index.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ To set up a Reactor, head over to our Portal and set up a new Stripe Reactor. If
 ## Using your new Reactor 
 <span class="base-alert warning">
   <span>
-    To use your Stripe Reactor, you'll need a <code>server-to-server</code> application with the following permissions <code>token:pci:read:low</code>, <code>token:pci:create</code> and <code>token:pci:use:reactor</code>.
+    To use your Stripe Reactor, you'll need a <code>server-to-server</code> application with the following permissions <code>token:pci:read:low</code>, <code>token:pci:create</code> and <code>token:pci:use:reactor</code>. <a href="https://portal.basistheory.com/applications/create?permissions=token%3Apci%3Aread%3Alow&permissions=token%3Apci%3Acreate&permissions=token%3Apci%3Ause%3Areactor&name=Card+Reactor" target="_blank">Click here to create one.</a>
```
Comment (koalaty-code) [inline collections/guides/collect_charge_cards_js/index.md:91]: The deep link to create an application is missing the `type` query param, if we care.

File: collections/guides/collect_pii_react/index.md (hunk 2 above)
Comment (djejaquino) [reply collections/guides/collect_pii_react/index.md:25]: This one has it. `elements`

**Pull request merged**